src/stratis/run.rs

```diff
@@ -48,7 +50,22 @@ async fn signal_thread() -> StratisResult<()> {
 /// Always check for devicemapper context.
 pub fn run(sim: bool) -> StratisResult<()> {
     if !sim {
-        unshare_namespace()?;
+        // If we are in a container, stratisd PID will be 1 and 'unshare' will not work. Calling
```

Comment:
Plz change this comment to, proposed:

"If stratisd is running as PID 1, then it is definitely in a container and the unshare_namespace() command method will fail, since the pid of the process will remain the same after the unshare command. Note that it is possible for stratisd to be running in a container and not as PID 1, but this
situation is less easily detected. stratisd will fail in the unshare_namespace() method if that is the case."

Please revise the comment if incorrect.

Reply:
Revised but kept the original intent. Thank you.